common/src/main/kotlin/web5/sdk/common/Convert.kt

```diff
@@ -46,7 +46,7 @@ public val B64URL_DECODER: Base64.Decoder = Base64.getUrlDecoder()
  * ```
  * // Example 1: Convert a ByteArray to a Base64Url encoded string without padding
  * val byteArray = byteArrayOf(1, 2, 3)
- * val base64Url = Convert(byteArray).toBase64Url(padding = false)
+ * val base64Url = Convert(byteArray).toBase64Url()
```

Comment:
should we put the padding = false back in the documentation ?

Reply:
i removed it coz padding is now false by default and added language around that default value, but can add back in just to show the usage?